### PR TITLE
gimp, horndis, osxfuse, tuntap migrated

### PR DESCRIPTION
**Pull request by AnastasiaSulyagina**
_In progress_

Formulae removed due to [deduplication](https://github.com/caskroom/homebrew-cask/issues/15603).

### CODE REVIEW

File: Formula/gimp.rb [hunk 1]
```diff
@@ -1,87 +0,0 @@
-class Gimp < Formula
```
Comment (xu-cheng) [inline Formula/gimp.rb]: Why do we want remove this formula? As far as I can tell, some users prefer to install software from homebrew even it's availabile from cask. The reasons can be preferring building from source, preferring homebrew's sandbox, preferring DIY build options, etc.

Comment (MikeMcQuaid) [inline Formula/gimp.rb]: @xu-cheng See the linked issue: https://github.com/caskroom/homebrew-cask/issues/15603

Comment (xu-cheng) [inline Formula/gimp.rb]: I expressed my thought on that issue.

Comment (AnastasiaSulyagina) [inline Formula/gimp.rb]: Thanks! 
As for this PR, it is not to be merged, it is just to see why do I have issues with debugging. So hopefully in the related issue we'll decide on what is finally to be deduplicated and I will do necessary migrations.

**Pull request closed without merging**